Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:20.19.5-slim AS development
```

Comment:
I'm pretty sure Node 20 is already EOL (or close to it), might be better to use a more recent version like 24 LTS